nerfstudio/viewer/export_panel.py

```diff
@@ -34,9 +34,11 @@ def _(_) -> None:
         populate_point_cloud_tab(server, control_panel, config_path)
     with server.add_gui_folder("Mesh"):
         populate_mesh_tab(server, control_panel, config_path)
+    with server.add_gui_folder("Splat"):
```

Comment:
@cvachha can we make this only appear when a splat model is loaded?

since this operation is fairly common it could also be moved above the point cloud / mesh exports I think!